axelar-chains-config/info/testnet.json

```diff
@@ -2427,14 +2305,15 @@
           ],
           "initialVerifierSetId": "3057ad91fc8e1de634e33ad33080d479b0614d0152b73b353e4dd676b3a66498",
           "address": "0xe432150cce91c13a887f7D836923d5597adD8E31",
-          "implementation": "0x05823c334150a48ACD5D325fBA16147c21bA3653",
-          "implementationCodehash": "0xa74b970bc244e333ce51fcf0e5cff6cf11a4f412be86d6088a8f230a32c5b1df",
-          "deploymentMethod": "create",
+          "implementation": "0xC9F7eC2963ea23fA292a801a2a6cea922eA69AF3",
```

Comment:
Ok, it's expected for implementation address to differ when using create2 since the domainSeparator is different between chains, which is a constructor arg: https://github.com/axelarnetwork/axelar-gmp-sdk-solidity/blob/main/contracts/gateway/AxelarAmplifierGateway.sol#L41